src/main/java/graphql/execution/ExecutionContextBuilder.java

```diff
@@ -131,6 +131,11 @@ public ExecutionContextBuilder subscriptionStrategy(ExecutionStrategy subscripti
         return this;
     }
 
+    /*
+     * @deprecated use {@link #graphQLContext(GraphQLContext)} instead
+     */
+    @Deprecated
+    @DeprecatedAt("2021-07-05")
```

Comment:
The context getter was deprecated on this date, for consistency, also deprecating the builder